lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -108,6 +108,17 @@ def visit_class(node)
 
       sig { override.params(node: SyntaxTree::Command).void }
       def visit_command(node)
+        if node.message.value == "test"
+          argument = node.arguments.parts.first
```

Comment:
We should add a test case for things that might break this. For example, what happens if an application has a `test` method that takes no arguments?

Reply:
Tests without a block are actually [valid](https://github.com/rails/rails/blob/1b10f5c04cf1c7bdf823c77780586e8f09e73c3f/activesupport/lib/active_support/testing/declarative.rb#L20-L22), it outputs as:

`No implementation provided for <test name>`

I think the intention is that you can use it as a reminder to write the test.

Reply:
Sure, but it doesn't mean that we should surface those as valid test methods either. Regardless, I don't see us doing a strict check on the count and type.of arguments being passed to the "test" call.